cime_config/acme/machines/template.case.run

```diff
@@ -5,12 +5,20 @@ template to create a case run script
 """
 
 import sys, os, calendar, shutil, time
+_CIMEROOT = os.environ.get("CIMEROOT")
```

Comment:
These lines need to be above the imports because they set the path to import from